tox.ini

```diff
@@ -17,6 +17,7 @@ deps =
     dj32: Django>=3.2,<4.0
     dj40: Django>=4.0,<4.1
     dj41: Django>=4.1,<4.2
+    dj42: Django>=4.2,<4.3
```

Comment:
There won't be a 4.3 according to Django's versioning policy, this should say `>=4.2,<5.0`.

Reply:
even doesn't know about that policy.